docker/scripts/run_connected_tests.sh

```diff
@@ -2,14 +2,15 @@
 gradle :testApp:assembleDebug
 gradle :android-sdk:assembleGrpcDebugAndroidTest
 
-gcloud config set project mobilecoin-60c
+gcloud config set project android-sdk-exper
```

Comment:
I had to change the GCloud project to a new one @holtzman created, since we couldn't figure out how to access the old one.